src/app/pages/system/update/manualupdate/manualupdate.component.ts

```diff
@@ -57,8 +57,11 @@ export class ManualUpdateComponent {
     {
       type: 'upload',
       name: 'filename',
-      placeholder: T('Update file to be installed'),
-      tooltip: T(''),
+      placeholder: T('Update File'),
+      tooltip: T('The file used to manually update the system. Browse to\
+                  the update file on the local system to upload and\
```

Comment:
Much improved, although it's hard to tell what the "local system" is.  Is that on the NAS, or on the computer running the browser?